### PR TITLE
media-libs/openimageio: remove 1.3.5, remove qt4 support, and v1.8.7

### PR DESCRIPTION
**Pull request by dracwyrm**
Removing qt4 support:
Closes: https://bugs.gentoo.org/644520

Add libressl support to 1.7.17:
Closes: https://bugs.gentoo.org/644638

@SoapGentoo Order of merge:
Remove old 1.3.5 version
Remove depreciated qt4 support
Version bump to 1.8.7

### CODE REVIEW

File: media-libs/openimageio/openimageio-1.8.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	ptex? ( media-libs/ptex:= )
+	python? (
+		${PYTHON_DEPS}
+		dev-libs/boost:=[python,${PYTHON_USEDEP}]
+	)
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+		opengl? ( dev-qt/qtopengl:4 )
```
Comment (SoapGentoo) [inline media-libs/openimageio/openimageio-1.8.7.ebuild]: `dev-qt/qtopengl:4` :face_with_head_bandage:

File: media-libs/openimageio/openimageio-1.8.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+RDEPEND=">=dev-libs/boost-1.62:=
+	dev-libs/pugixml:=
+	>=media-libs/ilmbase-2.2.0-r1:=
+	media-libs/libpng:0=
+	>=media-libs/libwebp-0.2.1:=
+	>=media-libs/openexr-2.2.0-r2:=
+	media-libs/tiff:0=
+	sys-libs/zlib:=
+	virtual/jpeg:=
```
Comment (SoapGentoo) [inline media-libs/openimageio/openimageio-1.8.7.ebuild]: `virtual/jpeg:0`

File: media-libs/openimageio/openimageio-1.8.7.ebuild [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+		-DUSE_NUKE=OFF
+		-DUSE_OCIO=$(usex colorio)
+		-DUSE_OPENCV=$(usex opencv)
+		-DUSE_OPENGL=$(usex opengl)
+		-DUSE_OPENJPEG=$(usex jpeg2k)
+		-DUSE_OPENSSL=$(usex ssl)
+		-DUSE_PTEX=$(usex ptex)
+		-DUSE_PYTHON=$(usex python)
+		-DUSE_QT=$(usex qt5)
+		-DUSE_SIMD="$(IFS=","; echo "${mysimd[*]}")"
```
Comment (SoapGentoo) [inline media-libs/openimageio/openimageio-1.8.7.ebuild]: there's a better way to do this, but I forgot :laughing:

Comment (dracwyrm) [reply media-libs/openimageio/openimageio-1.8.7.ebuild]: I thought this was the better way.

File: media-libs/openimageio/openimageio-1.8.7.ebuild (hunk 2 above)
Comment (dracwyrm) [reply media-libs/openimageio/openimageio-1.8.7.ebuild]: It was meant to be :0

File: media-libs/openimageio/openimageio-1.8.7.ebuild (hunk 1 above)
Comment (dracwyrm) [reply media-libs/openimageio/openimageio-1.8.7.ebuild]: I even added this line after I added the other 3 when double checking depends, as 1.8.x has a new depend on qtwidgets.

**Comment by monsieurp** [comment]
If everyone approves, let's go ahead then.

**Pull request closed without merging**